### PR TITLE
Add back Python unit tests.

### PR DESCRIPTION
**Pull request by MaikuMori**
That particular function didn't exist in the code anymore, I made a few
changes to apply it to the new variation of the function.

### CODE REVIEW

File: grapher/file_grapher_test.py [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
-    #             exp_module_name,
-    #             act_module_name,
-    #             msg=('{}: {} != {}'.format(filename, exp_module_name, act_module_name))
-    #         )
+    def test_module_path_to_parent_module_name(self):
+        """ Check if parent module name is correctly detected. """
+        cases = [
+            ('foo.py', ''),
+            ('foo/bar.py', 'foo'),
+            ('foo/__init__.py', ''),
```
Comment (beyang) [inline grapher/file_grapher_test.py]: should this map to `foo`?

File: grapher/file_grapher_test.py [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
+            ('foo/bar/__init__.py', 'foo'),
```
Comment (beyang) [inline grapher/file_grapher_test.py]: and this to `foo.bar`? I know there is technically a distinction between "module" and "package", but does the toolchain distinguish between the two? For instance, if we are constructing the full name of a function defined in `foo/bar/__init__.py`, we'd want it to be `foo.bar.function_name` and not `foo.function_name`, right?

Comment (MaikuMori) [inline grapher/file_grapher_test.py]: It's supposed to give you parent module/package name. The other part I obtain using jedi.

Here a picture with example source and output of this function + output I piece together using Jedi. The format is:
`module_path_to_parent_module_name_output|jedi_output`.

module_function

Function name is maybe a bit misleading. Using the foo.bar example. If there was class difined in the `__init__.py`, I would get `bar.SomeClass` from jedi and I need to get `foo` from this function to piece together correct path.

The relevant code is [here](https://github.com/sourcegraph/srclib-python/blob/add-back-unit-tests/grapher/file_grapher.py#L215), but there are different cases how I obtain name, in most simplest case it's just `jedi_def.full_name` which will return `bar.SomeClass` for the above example.

**Comment by beyang** [comment]
lgtm!

**Pull request merged**